dynawo/CMakeLists.txt

```diff
@@ -346,33 +338,38 @@ endif()
 find_package(libXML 0.2.4 REQUIRED PATHS ${LIBXML_HOME})
 
 # libIIDM
-find_package(libIIDM 0.2.8 REQUIRED PATHS ${LIBIIDM_HOME}
-  COMPONENTS
-    core
-    xml
-    ext-activeseason
-    ext-activeseason-xml
-    ext-busbarsectionposition
-    ext-busbarsectionposition-xml
-    ext-connectableposition
-    ext-connectableposition-xml
-    ext-currentlimitsperseason
-    ext-currentlimitsperseason-xml
-    ext-generatoractivepowercontrol
-    ext-generatoractivepowercontrol-xml
-    ext-hvdcangledroopactivepowercontrol
-    ext-hvdcangledroopactivepowercontrol-xml
-    ext-hvdcoperatoractivepowerrange
-    ext-hvdcoperatoractivepowerrange-xml
-    ext-generatorentsoecategory
-    ext-generatorentsoecategory-xml
-    ext-generatorstartup
-    ext-generatorstartup-xml
-    ext-loaddetail
-    ext-loaddetail-xml
-    ext-standbyautomaton
-    ext-standbyautomaton-xml
+if(CXX11_ENABLED)
+  find_package(LibXml2 REQUIRED PATHS ${LIBXML2_HOME})
```

Comment:
Please specify the minimum version.

Reply:
There is no minimum version for powsybl; I choose 2.9 by example

Reply:
Please put 2.9 so that we are coherent with 3rd party cmake files

Reply:
To be coherent I may put "include( libxml2.cmake)". That will be the good idea. With only "2.9" it does not work; it needs more changes

Reply:
No why include(libxml2.cmake) ? It has no reason being put there

Reply:
I wrote the file libxml2.cmake exacty to get rid of all logistics about a 3rd-package. Each time you need a target (in this case LibXml2::LibXml2) just include the file. In one line you get rid of names written sometimes with capitals, with versions, with different finders which are written in different directories, and uses different LIBXML2_HOME or ROOT or whatever. That file centralizes all this useless aspect wrapping the find_package call (which main purpose is to build the target). It makes thins a lot easier to maintain (version appears once for all file), to test (individual testing is easy while installing a 3rdParty), do document (we lack some). It is the hearth of thinking object: giving end parts (in this case the libxml2.cmake file) the abolity of taking responsabilities.

Reply:
Anyway, it is fixed in the last push